sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/FieldValue_internal.Serialization.cs

```diff
@@ -17,7 +18,7 @@ internal static FieldValue_internal DeserializeFieldValue_internal(JsonElement e
         {
             FieldValueType type = default;
             string valueString = default;
-            string valueDate = default;
+            DateTimeOffset? valueDate = default;
             string valueTime = default;
```

Comment:
Is Time support in now as well?

Reply:
It is :) 
although the changes to the swagger are still in PR. See https://github.com/Azure/azure-rest-api-specs/pull/8977